src/@sdk/react/components/CredentialsProvider/CredentialsProvider.tsx

```diff
@@ -10,7 +10,7 @@ export function CredentialsProvider({
   const [signIn] = useSignIn();
 
   const autoSignIn = async () => {
-    const credentials = await navigator.credentials.get({
+    const credentials = await (navigator.credentials as any).get({
```

Comment:
We should avoid specifying anything as any type. We decided to use typescript to have advantage of type checking as it allows to restrict possible types of data to those, which are handled in given part of code. It then might lead to having less bugs in an application, because unexpected usage of data doesn't occur. Please consider replacing it with specified type. I see adding `@types/webappsec-credential-management` as a dependency may solve the problem as it is referred in typescript issue https://github.com/microsoft/TypeScript/issues/34550

Reply:
Yes. i also find this link for solving it. but when i testing with this package i got an other issue that they can't found an other reference. more detail informations are will sharing  after testing again.